--- Running scipy.fft.idct#19YY/idctYY, strategy: relancer
[ERR MSG]: OrderedDict([('project', 'scipy.fft.idct#19YY'), ('notebook', 'idctYY'), ('text', '')])
[INFO] This case is fully fixed!


real	0m0.452s
user	0m2.111s
sys	0m8.868s
